docs/dafni.md

```diff
@@ -10,6 +10,11 @@ following:
 
 - In the `Data` tab:
   - Upload the input `YAML` file as a dataset.
+    Please note that if your `YAML` file contains `extensions` section, it should include
+    additional linux path strings as follows:
+    `extensions: [
+      /dafni/inputs/**.py
+    ]`
```

Comment:
Looks good, but I would be even more explicit indicating example extension files:

```suggestion
  - Upload the input `YAML` file as a dataset. Please note that if your `YAML` file 
  contains `extensions` section, it should include additional linux path strings as
  follows:

    ```
    extensions: [
      /data/inputs/extension_1.py,
      /data/inputs/extension_2.py,
      etc...
    ]
    ```

```